### PR TITLE
restrictions spelling and more

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

**Review by DavidBruchmann — COMMENTED** [review]
I deleted on comment, take care of it for the case that you got already mails with my comments.

**Review by franzholz — COMMENTED** [review]
I have removed all "you" talks.

File: Documentation/Developer/Index.rst [hunk 1]
```diff
@@ -15,8 +15,8 @@ on the link in the upper right corner in the modeler.
    You can enable/disable the advanced options with the access key "a"
 
 
-If you use the advanced options you should know what you do. There a many possibilities to run into misconfiguration,
-when using the advanced options. They are for special settings as they might be useful for the experienced TYPO3 developer.
+If you use the advanced options, you should know what you do. There a many possibilities to run into misconfiguration,
```
[the remark this answers is not in the document]
Comment (franzholz) [reply Documentation/Developer/Index.rst]: Commas are also used in English:

https://www.englisch-hilfen.de/grammar/kommasetzung.htm

File: Documentation/Developer/Index.rst [hunk 2]
```diff
@@ -59,44 +59,44 @@ Enter the basic meta data for your extension in the left panel of the modeler:
...
-+--------+--------------------------------------------------------+
-|**Name**|A short, speaking name that is displayed in the backend |
-+--------+--------------------------------------------------------+
-|**Key** |A unique key to identify the plugin in your extension   |
-+--------+--------------------------------------------------------+
++--------+---------------------------------------------------------+
+|**Name**| A short, speaking name that is displayed in the backend |
++--------+---------------------------------------------------------+
+|**Key** | A unique key to identify the plugin in your extension   |
++--------+---------------------------------------------------------+
```
Comment (DavidBruchmann) [inline Documentation/Developer/Index.rst:82]: Did you render the page? Additional signs shouldn't be required I think, as the lines are markup for rendering only.

**Review by franzholz — COMMENTED** [review]
alpha beta

**Comment by franzholz** [comment]
More improvements can be done.
However this should be done later. One pull request should not have too many modifications.

File: Documentation/Developer/Index.rst [hunk 3]
```diff
@@ -137,9 +137,9 @@ Each property has these fields:
 +-----------------+-----------------------------------------+----------------------------------------------------------------------------+
 
 
-The frontend form handling of the property types file and image are not yet implemented, due to a missing implementation in extbase.
-You have to implement the upload handling by yourself! The property select list has no configuration options in the modeler
-yet. You have to add the items list in the TCA configuration.
+The frontend form handling of the property types file and image are not yet implemented, due to a missing implementation in Extbase.
+The upload handling must be coded by the programmer! The property select list has no configuration options in the modeler
+yet. The items list must be inserted into the TCA configuration.
```
Comment (DavidBruchmann) [inline Documentation/Developer/Index.rst:143]: > The property select list has no configuration options in the modeler yet. The items list must be inserted into the TCA configuration.

This is not clear for me - before and after the edit.

**Review by DavidBruchmann — COMMENTED** [review]
> Otherwise it could have alpha or beta state.

this you did ignore, but it's not a big deal I guess.

**Comment by DavidBruchmann** [comment]
right, the change in `Documentation/Developer/Index.rst` I'd revers though before committing.

**Comment by DavidBruchmann** [comment]
If the rendering is wrong, then all tables had to be changed and you changed only one.

File: Documentation/Developer/Index.rst (hunk 2 above)
Comment (franzholz) [reply Documentation/Developer/Index.rst:82]: No, at the moment I have no working solution to check the rendering.
I prefer to have a leading blank. 
I do not want to change all tables.

File: Documentation/Developer/Index.rst (hunk 3 above)
Comment (franzholz) [reply Documentation/Developer/Index.rst:143]: The TCA can have a select box. The chosable items of the select box are coded in the TCA of this table.
However this documentation should not eplain these details.

**Comment by franzholz** [comment]
> If the rendering is wrong, then all tables had to be changed and you changed only one.

We can check first if this modification has any improvements or not.
Other tables can be changed later if this makes sense.

**Comment by DavidBruchmann** [comment]
You should have an indication (like docu for rst-documents) for those changes, so without one there is no reason for a change, especially as you never tested the rendering.

**Review by franzholz — COMMENTED** [review]
leading blank removed

**Review by DavidBruchmann — APPROVED** [review]

**Pull request merged**